app/src/server/scripts/sendInvoiceEmailToExistingClients.ts

```diff
@@ -0,0 +1,46 @@
+import { prisma } from "../db";
+import { sendInvoiceNotificationWithoutRequiredPayment } from "../emails/sendInvoiceNotificationWithoutRequiredPayment";
+import { sendToAdmins } from "../emails/sendToAdmins";
+
+console.log("Doing: sending emiails");
+
+//Neagative credits mean that the servcie was used
+const creditAdjustments = await prisma.creditAdjustment.findMany({
+  where: {
+    amount: {
+      lt: 0,
+    },
+  },
+  include: {
+    invoice: true,
+    project: true,
+  },
+});
+
+for (const creditAdjustment of creditAdjustments) {
```

Comment:
We have a lot of invoices with $0 spent. We do not know if it had any usage, or if it is just an empty invoice since we issued invoices for all the projects, even without any usage. Cycling through the negative credit adjustments allows finding only invoices with some usage.

Reply:
Are we sure that there is currently only one creditAdjustment for each project?

Reply:
Yes, each project should have only one invoice at this moment, and only one negative credit adjustment associated with it.